src/Microsoft.Azure.WebJobs.ServiceBus/WebJobs.ServiceBus.csproj

```diff
@@ -9,8 +9,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.ServiceBus" Version="2.0.0" />
-    <PackageReference Include="StyleCop.Analyzers" Version="1.1.0-beta004">
+    <PackageReference Include="Microsoft.Azure.ServiceBus" Version="3.0.2" />
```

Comment:
@mathewc / @fabiocav -- this change is bumping our ServiceBus dependency. Do we have any concerns with that? I know we wanted to do it anyway, but wanted to make sure it was clear that it was happening here.